From weiyi.huang@gmail.com Sat Aug 16 16:51:45 2008
From: Huang Weiyi <weiyi.huang@gmail.com>
Date: Sun, 17 Aug 2008 07:51:23 +0800
Subject: USB: removed unused #include <version.h>
To: gregkh@suse.de, greg@kroah.com
Cc: linux-usb@vger.kernel.org
Message-ID: <20080817073231.1634.WEIYI.HUANG@gmail.com>


The driver(s) below do not use LINUX_VERSION_CODE nor KERNEL_VERSION.
  drivers/usb/atm/ueagle-atm.c
  drivers/usb/gadget/amd5536udc.c
  drivers/usb/misc/iowarrior.c
  drivers/usb/serial/garmin_gps.c

This patch removes the said #include <version.h>.

Signed-off-by: Huang Weiyi <weiyi.huang@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 drivers/usb/atm/ueagle-atm.c    |    1 -
 drivers/usb/gadget/amd5536udc.c |    1 -
 drivers/usb/misc/iowarrior.c    |    1 -
 drivers/usb/serial/garmin_gps.c |    1 -
 4 files changed, 4 deletions(-)

--- a/drivers/usb/atm/ueagle-atm.c
+++ b/drivers/usb/atm/ueagle-atm.c
@@ -64,7 +64,6 @@
 #include <linux/ctype.h>
 #include <linux/sched.h>
 #include <linux/kthread.h>
-#include <linux/version.h>
 #include <linux/mutex.h>
 #include <linux/freezer.h>
 
--- a/drivers/usb/gadget/amd5536udc.c
+++ b/drivers/usb/gadget/amd5536udc.c
@@ -44,7 +44,6 @@
 #include <linux/module.h>
 #include <linux/pci.h>
 #include <linux/kernel.h>
-#include <linux/version.h>
 #include <linux/delay.h>
 #include <linux/ioport.h>
 #include <linux/sched.h>
--- a/drivers/usb/misc/iowarrior.c
+++ b/drivers/usb/misc/iowarrior.c
@@ -19,7 +19,6 @@
 #include <linux/slab.h>
 #include <linux/sched.h>
 #include <linux/poll.h>
-#include <linux/version.h>
 #include <linux/usb/iowarrior.h>
 
 /* Version Information */
--- a/drivers/usb/serial/garmin_gps.c
+++ b/drivers/usb/serial/garmin_gps.c
@@ -38,7 +38,6 @@
 #include <linux/usb.h>
 #include <linux/usb/serial.h>
 
-#include <linux/version.h>
 
 /* the mode to be set when the port ist opened */
 static int initial_mode = 1;
